src/include/miopen/solver/implicitgemm_ck_util.hpp

```diff
@@ -114,15 +134,36 @@ ConvSolution InitInvokerFactoryNHWC(const ExecutionContext&,
     }
 
     ConvSolution result;
-    result.invoker_factory =
-        [ck_args     = CKArgsType{problem},
-         sh_conv_ptr = std::shared_ptr{std::move(*ptr_iter)}](const std::vector<Kernel>&) mutable {
-            return [ck_args = std::move(ck_args), sh_conv_ptr = std::move(sh_conv_ptr)](
-                       const Handle& handle, const AnyInvokeParams& primitive_parameters) {
-                const auto& data_ctx = primitive_parameters.CastTo<CastType>();
-                auto argument_ptr    = ck_args.MakeArgPtr(sh_conv_ptr, data_ctx.tensors);
-                auto invoker_ptr     = sh_conv_ptr->MakeInvokerPointer();
+    result.invoker_factory = [ck_args     = CKArgsType{problem},
+                              sh_conv_ptr = std::shared_ptr{std::move(*ptr_iter)}](
+                                 const std::vector<Kernel>&) mutable {
```

Comment:
@iq136boy This is formatting-only change. The "Clang Format" CI stage should fail, but it didn't. Why?

Reply:
@atamazov To be honest, I have no idea. I have called clang-format to set the format for the modified file locally before I push it to the github. And the following clang-format command get empty. 
clang-format-12 -style=file ../src/include/miopen/solver/implicitgemm_ck_util.hpp | diff - ../src/include/miopen/solver/implicitgemm_ck_util.hpp
Which means the "clang format" should be good?

Reply:
@iq136boy Likely so unless your clang-format tool or ./.clang-format file is corrupt or something else. Please check what CI prints during the "Clang Format" stage. This is important.

Reply:
(continued at https://github.com/ROCm/MIOpen/pull/2770#discussion_r1509099090)